### ISSUE TITLE
Indented Numbered List re-sorts when deleting item above it in hierarchy

### ISSUE THREAD

**Issue by vultureman** [MicrosoftDocs/msteams-docs#787]
Running Teams Version 1.2.00.17057 (64-bit) on Win10 1809.

In the Wiki feature for Teams, I've run into an issue with indented numbered lists. Please note that, in the example below, "Burritos" and everything below it is a level **under** "Mexican Food", not at the same level as "Mexican Food":

```
1. Mexican Food
    1. Tacos
        1. Types of Tacos
            1. Al pastor
            2. Tacos arabes
            3. Barbacoa
            4. Birria
        2. Taco History
    2. Burritos
    3. Chilaquiles
    4. Tostadas
    5. Enchiladas
```
If I hit backspace once at the beginning of the "Types of Tacos" line (put cursor before the "T" and hit backspace once), this should move the "Types of Tacos" text up onto the "Tacos" line above it. It does, but the items that were previously nested underneath "Types of Tacos" re-sort in reverse order, which isn't expected behavior.
```
1. Mexican Food
    1. TacosTypes of Tacos
        1. Birria
        2. Barbacoa
        3. Tacos arabes
        4. Al pastor
        5. Taco History
    2. Burritos
    3. Chilaquiles
    4. Tostadas
    5. Enchiladas
```
In my mind, there are 2 issues:

1. The reverse re-sort of the list is definitely a bug that needs to be resolved. Nobody should expect this.
2. I guess I expected formatting behavior to be similar to what Word does, which I've attached as a comparison with screenshots. In Teams, I guess I would have expected the one backspace to do the equivalent of the 3 backspaces in Word, and moved the "Types of Tacos" text up to the "Tacos" line, but left the nested text at its nested level four tabs deep. I was using Word as a part of the Office 365 ProPlus suite, and it was Version 1906 in the Monthly Channel (Build 11727.20230 Click-to-Run).

![tacos test - teams and word](https://user-images.githubusercontent.com/3814145/61082040-a6f70680-a3f6-11e9-915f-c204f1e97da1.png)

tl;dr - delicious tacos are re-sorted in reverse order when deleting the parent numbered line

**Comment by Gousia-MSFT** [comment]
@vultureman Thanks for reporting this. We have raised a bug for this and we are looking into it.
